src/config/networks.ts

```diff
@@ -37,5 +37,12 @@ export const defaultNetworks: NetworksConfig = {
     proxyFactoryAddress: '0xfC7577774887aAE7bAcdf0Fc8ce041DA0b3200f7',
     multiSendAddress: '0x8D29bE29923b68abfDD21e541b9374737B49cdAD',
     fallbackHandlerAddress: '0x40A930851BD2e590Bd5A5C981b436de25742E980'
+  },
+  // xdai
+  100: {
+    masterCopyAddress: '0x6851D6fDFAfD08c0295C392436245E5bc78B0185',
+    proxyFactoryAddress: '0xfC7577774887aAE7bAcdf0Fc8ce041DA0b3200f7',
+    multiSendAddress: '0x8D29bE29923b68abfDD21e541b9374737B49cdAD',
+    fallbackHandlerAddress: '0xd5D82B6aDDc9027B22dCA772Aa68D5d74cdBdF44'
```

Comment:
Why do we not use the official fallback handler on xdai?

Reply:
Is not this the right one? https://github.com/gnosis/safe-contracts/blob/v1.2.0/networks.json#L118
From where did we get the fallback handler at `0x40A930851BD2e590Bd5A5C981b436de25742E980` for the other networks?

Reply:
so this seems to be the 1.1.1 fallback handler and the other is 1.1.0 (as mentioned on etherscan). Note 1.1.1 is the current. Code wise 1.1.0 and 1.1.1 should be the same.

See https://etherscan.io/address/0x40A930851BD2e590Bd5A5C981b436de25742E980
and https://etherscan.io/address/0xd5D82B6aDDc9027B22dCA772Aa68D5d74cdBdF44